styx-api-service/src/test/java/com/spotify/styx/api/WorkflowResourceTest.java

```diff
@@ -113,8 +114,10 @@
   @Mock private WorkflowActionAuthorizer workflowActionAuthorizer;
   @Mock private GoogleIdToken idToken;
   @Mock private RequestAuthenticator requestAuthenticator;
+  @Mock private Clock clock;
 
   private static final String SERVICE_ACCOUNT = "foo@bar.iam.gserviceaccount.com";
+  private static final Instant deploymentTime = Instant.ofEpochSecond(0);
```

Comment:
maybe use another time/date here?

Reply:
The thing is that the epoc is the more probable value to be used as default value so using something else is better. You can use even Instant.now()